site/docs/visual_elements/favicon.mdx

```diff
@@ -0,0 +1,161 @@
+---
+name: Favicon
+route: /visual-assets/favicon/
+menu: Visual assets
+---
+
+import LargeParagraph from "../../src/components/LargeParagraph";
+import Image from "../../src/components/Image";
+import Link from "../../src/components/Link";
+
+# Favicon
+
+<LargeParagraph>The City of Helsinki favicon is used in digital services to identify the site window or tab.</LargeParagraph>
+
+<p>The favicon consists of the text <em>Helsinki</em> framed by the shape derived from the city crest. The favicon must be used in accordance with the <Link href="#" external>Visual Identity Guidelines - Favicon</Link>.</p>
+
+**Do not attempt to recreate the favicon!** 
+Always use the appropriate original file. 
+
+**Note:** The City of Helsinki logo is a registered trademark.
+
+## Principles 
+Official City of Helsinki services must always show the City of Helsinki favicon: 
+- sitewide, in the browser website title
+- use the same favicon variant consistently
+
+## Usage
+
+### Contrast
+The Helsinki framed favicon is primarily used in black, white, or with a transparent background depending on the circumstance.
+- Check that the colour contrast of the favicon and its background is compliant with <Link href="https://www.w3.org/TR/WCAG21/#contrast-minimum" external>AA Level contrast ratios</Link>.
+- For more information, see the <Link href="https://brand.hel.fi/en/colours/" external>Visual Identity Guidelines - Colours</Link>.
+
+### Download
+
+<p id="github">The favicon of City of Helsinki digital services is a minimised version of the Helsinki logo. Different favicons are available as a downloadable ZIP package.</p>
+<p>You can <Link href="https://github.com/City-of-Helsinki/helsinki-design-system/releases/latest/download/hds-favicon-kit.zip" external>download the latest HDS Favicon kit from the HDS GitHub repository</Link>.</p>
+
+## Installation
+First, use the download link to download the files listed below. Unzip the archive to the root directory of your website.
+
+```android-chrome-192x192.png```
```

Comment:
Could we improve this example? Some ideas:
- We do not need to list all of these favicon files here. They are included in the ZIP package so the developer does not need to worry that they have all the necessary files.
- Add clearer instructions on what to do with the ZIP. Something numbered like; "1. Download the ZIP package, 2. Extract it to the root directory of the website, 3. Copy the following tags to the HTML head" ...

Reply:
Should I remove the list alltogether, or…?